modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ImportTaskResourcePerformanceTest.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.batch.engine.resource.v1_0.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.headless.batch.engine.client.dto.v1_0.ImportTask;
+import com.liferay.headless.batch.engine.client.http.HttpInvoker;
+import com.liferay.headless.batch.engine.client.serdes.v1_0.ImportTaskSerDes;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.test.randomizerbumpers.UniqueStringRandomizerBumper;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.LinkedHashMapBuilder;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.io.Closeable;
+
+import java.util.Map;
+import java.util.Objects;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Vendel Toreki
+ */
+@RunWith(Arquillian.class)
+public class ImportTaskResourcePerformanceTest
+	extends BaseTaskResourcePerformanceTestCase {
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		BaseTaskResourcePerformanceTestCase.setUpClass();
+
+		_jsonTemplates = LinkedHashMapBuilder.put(
+			"com.liferay.headless.admin.user.dto.v1_0.UserAccount",
+			_createUserAccountJSONTemplate()
+		).build();
+	}
+
+	@Test
+	public void testPostImportTaskWithUserAccount() throws Exception {
+		_testPostImportTask(
+			"com.liferay.headless.admin.user.dto.v1_0.UserAccount");
+	}
+
+	private static String _createUserAccountJSONTemplate() throws Exception {
+		return JSONUtil.put(
```

Comment:
Maybe in the future, we could refactor the way we generate the input data, to move it to files for each class name (UserAccount, etc), and instead of using constants (such as `"[$ALTERNATE_NAME$]"`), we could use something like `[$RANDOM_STRING(3)$]`, etc, but just for a possible future idea

Reply:
Yes, that can work. But it will make the JSON generation more complicated. Now it's just a simple string replace. I think we can start thinking of generalizing the JSON generation more when we include new entities in the batch performance test.
Probably even execute a smaller export, just to get a JSON sample, and then multiply and automatically replace some variables to generate a larger file.